python/storage/kserve_storage/kserve_storage.py

```diff
@@ -55,6 +56,13 @@
 _HDFS_SECRET_DIRECTORY = "/var/secrets/kserve-hdfscreds"
 _HDFS_FILE_SECRETS = ["KERBEROS_KEYTAB", "TLS_CERT", "TLS_KEY", "TLS_CA"]
 
+# Azure async download configuration
+_AZURE_MAX_FILE_CONCURRENCY = int(os.getenv("AZURE_MAX_FILE_CONCURRENCY", "4"))
```

Comment:
I assumed currently, there is no way to adjust these parameters?

Reply:
Thanks @lizzzcai, I didn't know about ClusterStorageContainer.
Can you approve https://github.com/kserve/kserve/pull/4709 then? I moved it over there because of a probably rushed decision when finding myself in a "github entanglement".